<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="plot_tabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="event_tab">
         <attribute name="title">
          <string>Event plot</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="PlotWidget" name="event_plot"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="reset_event_axes_button">
              <property name="text">
               <string>Reset axes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="toggle_guides_button1">
              <property name="text">
               <string>Toggle guides</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="toggle_markslines_button1">
              <property name="text">
               <string>Toggle marks/lines</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="spectrum_tab">
         <attribute name="title">
          <string>Spectrum plot</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="PlotWidget" name="spectrum_plot"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="reset_spectrum_axes_button">
              <property name="text">
               <string>Reset axes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="toggle_guides_button2">
              <property name="text">
               <string>Toggle guides</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="toggle_markslines_button2">
              <property name="text">
               <string>Toggle marks/lines</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="clear_run_button">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="single_button">
          <property name="text">
           <string>Single</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="run_stop_button">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Run time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="run_time_label">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0 s</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="num_events_label">
          <property name="text">
           <string>(0 events)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,0,0">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Range:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="range_box"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Offset:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="offset_box">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Trigger threshold:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="threshold_box">
          <property name="suffix">
           <string> V</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-40.000000000000000</double>
          </property>
          <property name="maximum">
           <double>40.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Enable trigger:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="trigger_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Trigger on:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="trigger_channel_box">
          <item>
           <property name="text">
            <string>Channel A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Channel B</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Pulse polarity:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="polarity_box"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Timebase:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSpinBox" name="timebase_box">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Sampling time:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="sampling_time_label">
          <property name="text">
           <string>0 μs</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Pre-trigger window:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QDoubleSpinBox" name="pre_trigger_box">
          <property name="suffix">
           <string> μs</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Post-trigger window:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QDoubleSpinBox" name="post_trigger_box">
          <property name="suffix">
           <string> μs</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Number of samples:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="num_samples_label">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Number of captures:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QSpinBox" name="num_captures_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Baseline correction:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QCheckBox" name="baseline_correction_box">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Show channel A:</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QCheckBox" name="ch_A_enabled_box">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Show channel B:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QCheckBox" name="ch_B_enabled_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Lower level discriminator:</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QDoubleSpinBox" name="lld_box">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Upper level discriminator:</string>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="QDoubleSpinBox" name="uld_box">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="value">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Histogram bins:</string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QSpinBox" name="num_bins_box">
          <property name="maximum">
           <number>4095</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QCheckBox" name="upper_trigger_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Enable upper threshold</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QDoubleSpinBox" name="upper_threshold_box">
          <property name="suffix">
           <string> V</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-40.000000000000000</double>
          </property>
          <property name="maximum">
           <double>40.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Upper threshold (SW):</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Run duration:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="run_duration_box">
          <property name="suffix">
           <string> s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>600</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
